pkg/ccl/logictestccl/testdata/logic_test/tenant_capability

```diff
@@ -39,6 +39,7 @@ can_check_consistency      false
 can_debug_process          false
 can_use_nodelocal_storage  false
 can_view_all_metrics       false
+can_view_all_metrics       false
```

Comment:
Was this supposed to be `can_view_network_info`?  I kinda wonder if we could re-use `can_debug_process` for this endpoint.  The latency of the node we are connecting to is _kinda_ a property of the process so it makes some sense. Thoughts?